plugins/org.eclipse.osee.disposition.rest/src/org/eclipse/osee/disposition/rest/internal/importer/DispoSetCopier.java

```diff
@@ -48,23 +48,27 @@ public DispoSetCopier(DispoConnector connector) {
       this.connector = connector;
    }
 
-   public List<DispoItem> copyAllDispositions(Map<String, Set<DispoItemData>> nameToDestItems,
+   public List<DispoItem> copyAllDispositionsSameSet(Map<String, Set<DispoItemData>> nameToDestItems,
       Collection<DispoItem> sourceItems, boolean isCoverageCopy, HashMap<String, String> reruns,
       OperationReport report) {
       return copyAllDispositions(nameToDestItems, sourceItems, isCoverageCopy, reruns, false, Collections.emptySet(),
-         report);
+         true, report);
    }
 
    public List<DispoItem> copyAllDispositions(Map<String, Set<DispoItemData>> nameToDestItems,
       Collection<DispoItem> sourceItems, boolean isCoverageCopy, HashMap<String, String> reruns,
-      boolean allowOnlyValidResolutionTypes, Set<String> validResolutionsTypes, OperationReport report) {
+      boolean allowOnlyValidResolutionTypes, Set<String> validResolutionsTypes, boolean isSameSet,
+      OperationReport report) {
       List<DispoItem> modifiedItems = new ArrayList<>();
 
       // Iterate through every source item since we want to try to find a match for every item in the source
       for (DispoItem sourceItem : sourceItems) {
          DispoItemData destItem = getCorrespondingDestItem(nameToDestItems, sourceItem);
```

Comment:
could this just return a sentinel?  instead of you having to check for null?

Reply:
Yes! I added a Sentinel for that and haven't added it everywhere yet. I just revised it ty.